.github/workflows/ruby.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build the docker-compose stack
+      run: docker-compose -f docker-compose-ci.yml up -d
```

Comment:
if all we need is redis, then lets use github actions service? No need for this docker stuff. 

The reason you see this in Jupiter is we have a very weird dependency with Solr. We need our code checkout, which we push config into solr, but if we use services, we don't have access to our code. Basically chicken and egg problem.

Here you can just use this:

```
jobs:
  test:
    runs-on: ubuntu-latest

    services:
      redis:
        image: redis:alpine
        ports: ["6379:6379"]
 steps:
    - uses: actions/checkout@v2
 ....etc...
```

Reply:
Thanks! This is definitely and improvement.